doc/adrs/161_add_support_for_undo_redo_on_semantic_changes_in_representations.adoc

```diff
@@ -0,0 +1,185 @@
+= [ADR-161] Add support for undo redo on semantic changes in representations
```

Comment:
Don't mention representations, it's only for semantic changes regardless of how they have been modified.